eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -398,3 +417,8 @@ jobs:
       inputs:
         artifactName: docs
         path: $(Build.ArtifactStagingDirectory)/docs
+
+    - task: PublishPipelineArtifact@1
```

Comment:
@scbedd after our last conversation any thoughts on how we might want to structure the artifact so there is only one artifact per package?

Reply:
In C this is not a problem. In C++ I nest these within folders: https://github.com/Azure/azure-sdk-for-cpp/pull/576